src/OnlineSales/Controllers/ImagesController.cs

```diff
@@ -5,6 +5,7 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.StaticFiles;
 using Microsoft.EntityFrameworkCore;
+using Nest;
```

Comment:
@WASandaruwan, what is the point to have Nest library included here? I do not see that you do anything with Elasticserch in the code of ImageController.cs ...